compiler/rustc_resolve/src/ident.rs

```diff
@@ -1546,6 +1593,7 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                     ident,
                     ns,
                     parent_scope,
+                    used_import,
```

Comment:
The import should always be `None` here, and `fn resolve_ident_in_lexical_scope` doesn't need the import argument.
`ribs.is_some()` means `used_import` is `None`, you can add an assert.

Reply:
I will remove `used_import` in `resolve_ident_in_lexical_scope`.